oemoflex/preprocessing.py

```diff
@@ -1218,3 +1220,63 @@ def create_electricity_bev_profiles(data_raw_path, data_preprocessed_path):
         profile_df.to_csv(
             os.path.join(data_preprocessed_path, 'sequences', k + '_profile.csv')
         )
+
+
+def create_profiles(exp_path, select_components):
+
+    def normalize_year(profile_df):
+        yearly_amount = profile_df.sum(axis=0)
+        profile_df = profile_df.divide(yearly_amount)
+        return profile_df
+
+    recalculation_functions = {
+        'normalize_year': normalize_year
+    }
+
+    oemof_tabular_settings_filepath = os.path.join(module_path, 'oemof-tabular-settings.yml')
+    with open(oemof_tabular_settings_filepath, 'r') as settings_file:
+        settings = yaml.safe_load(settings_file)
+        sequences_dir = settings['sequences-dir']
+        profile_file_suffix = settings['profile-file-suffix']
+        profile_name_suffix = settings['profile-name-suffix']
+
+    mapping_filepath = os.path.join(module_path, 'mapping-input-timeseries.yml')
+    with open(mapping_filepath, 'r') as mapping_file:
+        map = yaml.safe_load(mapping_file)
+
+    raw_path = exp_path.data_raw
+    preprocessed_path = exp_path.data_preprocessed
+
+    for component in select_components:
+
+        try:
```

Comment:
This try-except spans much too far to be helpful. If it fails, you know that the error occured somewhere in the next ~20 lines.

Reply:
Thank you! Especially for wrong recalculation function names in the yaml another KeyError would be hidden.